docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -132,26 +132,135 @@ __Example__:
     platformName: "Windows 10"
 ```
 
-### `testMatch`
+### `screenResolution`
 
-__Description__: The explicit name, regex, or location of the test files.
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Type__: *string* | *regex*
+__Description__: Set browser window screen resolution.
+
+__Type__: *string*
 
 __Example__:
 ```yaml
-    testMatch: '**/*.js'
+    screenResolution: "1920x1080"
 ```
 
-### `screenResolution`
+### `disableScreenshots`
+__Description__: Prevents TestCafe from taking screenshots. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#disablescreenshots).
 
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
+__Type__: *boolean*
 
-__Description__: Set browser window screen resolution.
+__Example__:
+```yaml
+  disableScreenshots: true
+```
+
+### `tsConfigPath`
+__Description__: The absolute or relative path to the TypeScript configuration file. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
```

Comment:
Is "current directory" the same as the `rootDir` setting?

Reply:
It should be different. This config field should be related to typescript config file while rootDir is for the whole testing project.